Trying:
    print_square = __import__('4-print_square').print_square
Expecting nothing
ok
Trying:
    print_square(0)
Expecting nothing
ok
Trying:
    print_square(3)
Expecting:
    ###
    ###
    ###
ok
Trying:
    print_square(4.5)
Expecting:
    Traceback (most recent call last):
    TypeError: size must be an integer
ok
Trying:
    print_square("World")
Expecting:
    Traceback (most recent call last):
    TypeError: size must be an integer
ok
Trying:
    print_square(None)
Expecting:
    Traceback (most recent call last):
    TypeError: size must be an integer
ok
Trying:
    print_square()
Expecting:
    Traceback (most recent call last):
**********************************************************************
File "./tests/4-print_square.txt", line 30, in 4-print_square.txt
Failed example:
    print_square()
Exception raised:
    Traceback (most recent call last):
      File "/usr/lib/python3.12/doctest.py", line 1361, in __run
        exec(compile(example.source, filename, "single",
      File "<doctest 4-print_square.txt[6]>", line 1, in <module>
        print_square()
    TypeError: print_square() missing 1 required positional argument: 'size'
**********************************************************************
1 items had failures:
   1 of   7 in 4-print_square.txt
7 tests in 1 items.
6 passed and 1 failed.
***Test Failed*** 1 failures.
